README.md

```diff
@@ -1,742 +1,77 @@
-
-childSpan.End();
-parentSpan.End();
-```
-
-#### Implicit parent propagation and assignment
-
-```csharp
-
-// calling StartActiveSpan starts a span and puts parentSpan into the ambient context
-// that flows in async calls.   When child is created, it implicitly becomes child of current span
-using (tracer.StartActiveSpan("parent span", out _))
-{
-    var childSpan = tracer.StartSpan("child span");
-
-    childSpan.End();
-}
-
-// parent span is ended when StartActiveSpan result is disposed
-```
-
-### Span with attributes
-
-Attributes provide additional context on span specific to specific operation it tracks such as HTTP/DB/etc call properties.
-
-```csharp
-// spans have Client, Server, Internal, Producer and Consumer kinds to help visualize them
-var span = tracer.StartSpan("span with attributes", SpanKind.Client);
-
-// attributes specific to the call
-span.SetAttribute("db.type", "redis");
-span.SetAttribute("db.instance", "localhost:6379[0]");
-span.SetAttribute("db.statement", "SET");
-span.End();
-```
-
-### Span with links
-
-[Links](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md#links-between-spans) allow to create relationships between different traces i.e. allow spans to have multiple relatives. They are typically used to trace batching scenarios where multiple traces are merged into another one.
-
-Links affect sampling decision and should be added before sampling decision is made (i.e. before span starts).
-
-```csharp
-SpanContext link1 = ExtractContext(eventHubMessage1);
-SpanContext link2 = ExtractContext(eventHubMessage2);
-
-var span = tracer.StartSpan("span with links", SpanKind.Server, DateTime.UtcNow, new [] {link1, link2});
-span.End();
-```
-
-### Span with events
-
-Events are timed text (with optional attributes) annotations on the span. Events can be added to current span (or any running span).
-
-```csharp
-using (tracer.StartActiveSpan("incoming HTTP request", SpanKind.Server, out var span))
-{
-    span.AddEvent("routes resolved");
-}
-
-// span is ended when StartActiveSpan result is disposed
-```
-
-### Context propagation out of process
-
-Examples will be added once [Global Propagators API is added.](https://github.com/open-telemetry/opentelemetry-dotnet/issues/581)
-
-### Auto-adapter implementation for Activity/DiagnosticSource
-
-`System.Diagnostics.Activity` is similar to OpenTelemetry Span. HttpClient, ASP.NET Core, SqlClient, & Azure SDKs use them to expose diagnostics events and context.
-
-Leaving aside subscription mechanism, here is an example how you may implement callbacks for Start/Stop Activity
-
-```csharp
-void StartActivity()
-{
-    tracer.StartActiveSpanFromActivity ("GET api/values", Activity.Current, out var span);
-
-    // extract other things from Activity and set them on span (tags to attributes)
-    // ...
-}
+The .NET [OpenTelemetry](https://opentelemetry.io/) client.

```

Comment:
```suggestion
The .NET [OpenTelemetry](https://opentelemetry.io/) SDK.

```

I think client may be a confusing term here

Reply:
I remember we've discussed before and the conclusion is to use client? For example:
https://github.com/open-telemetry/opentelemetry-go
https://github.com/open-telemetry/opentelemetry-python

Reply:
thanks! makes sense